x-pack/plugins/cross_cluster_replication/public/app/components/follower_index_form.js

```diff
@@ -247,14 +257,26 @@ export const FollowerIndexForm = injectI18n(
        * Remote Cluster
        */
       const renderRemoteClusterField = () => {
```

Comment:
It helps by grouping functions by behavior. Instead of heaving loads of functions declared at the component level. I like to toggle all first level methods when I open a component I never worked on and see the moving parts. If `renderForm()` needs 5 functions to make it easier to reason about, I don't really need that information when I first open a component file.